tools/translations/process_source_ts_files.py

```
@@ -108,67 +113,106 @@ def write_ts_file(target_file, root):
+
+                if filename not in locations:
+                    locations[filename] = location
+                    count = 1
+                else:
+                    message.remove(location)
+                    location = locations[filename]
+                    count = int(location.get('line')) + 1
+
+                location.set('line', str(count))
+
+            bytes = source.findall('byte')
```

Ruff (0.16.1)

[error] Variable `bytes` is shadowing a Python builtin

(A001)